src/python/pants/bin/local_pants_runner.py

```diff
@@ -257,7 +257,9 @@ def _maybe_handle_help(self):
     if self._options.help_request:
       help_printer = HelpPrinter(self._options)
       result = help_printer.print_help()
-      self._exiter(result)
+      return result
+    else:
```

Comment:
I don't think we need `else: return None` it is implicit.

I think what we actually need to return a is a the boolean value of `self._options.help_request` and not result.